playwright/src/main/java/com/microsoft/playwright/junit/Options.java

```diff
@@ -0,0 +1,112 @@
+package com.microsoft.playwright.junit;
+
+import com.microsoft.playwright.APIRequest;
+import com.microsoft.playwright.Browser;
+import com.microsoft.playwright.BrowserType;
+import com.microsoft.playwright.Playwright;
+import com.microsoft.playwright.options.ViewportSize;
+
+import java.nio.file.Path;
+
+public class Options {
+  private String baseUrl;
+  private Path storageStatePath;
+  private ViewportSize viewportSize;
+  private String channel;
+  private boolean headless;
```

Comment:
Let's make it `Boolean` to make it clear whe `LaunchOptions.headless` should not be overridden (similar to how you check `channel`).